pkg/gateway/model_build_targetgroup.go

```diff
@@ -28,48 +27,56 @@ type TargetGroupModelBuilder interface {
 	Build(ctx context.Context, srvExport *mcs_api.ServiceExport) (core.Stack, *latticemodel.TargetGroup, error)
 }
 
-type targetGroupBuilder struct {
-	client.Client
-
+type TargetGroupBuilder struct {
+	log           gwlog.Logger
+	client        client.Client
 	serviceExport *mcs_api.ServiceExport
 	Datastore     *latticestore.LatticeDataStore
 	cloud         lattice_aws.Cloud
-
-	defaultTags map[string]string
+	defaultTags   map[string]string
 }
 
 // triggered from serviceexport
-func NewTargetGroupBuilder(client client.Client, datastore *latticestore.LatticeDataStore, cloud lattice_aws.Cloud) *targetGroupBuilder {
-	return &targetGroupBuilder{
-		Client:    client,
+func NewTargetGroupBuilder(
+	log gwlog.Logger,
+	client client.Client,
+	datastore *latticestore.LatticeDataStore,
+	cloud lattice_aws.Cloud,
+) *TargetGroupBuilder {
+	return &TargetGroupBuilder{
+		log:       log,
+		client:    client,
 		Datastore: datastore,
 		cloud:     cloud,
 	}
 }
 
 type targetGroupModelBuildTask struct {
-	client.Client
+	log           gwlog.Logger
+	client        client.Client
 	serviceExport *mcs_api.ServiceExport
 	targetGroup   *latticemodel.TargetGroup
 	tgByResID     map[string]*latticemodel.TargetGroup
 	stack         core.Stack
-
-	Datastore *latticestore.LatticeDataStore
-	cloud     lattice_aws.Cloud
+	Datastore     *latticestore.LatticeDataStore
```

Comment:
For consistency, I like datastore, but I see places where latticeDataStore is used, also it should be private property.

If it's not a burden can you align datastores in your PR in places where you change it. Not asking to go beyond that.

Reply:
Yup, done next commit